test/utils/permalinks/Permalinks-test.js

```diff
@@ -34,6 +34,7 @@ function mockRoom(roomId, members, serverACL) {
 
     return {
         roomId,
+        getCanonicalAlias: () => roomId,
```

Comment:
we should probably return a real alias instead of a room ID, otherwise the tests could become confusing